### PR TITLE
Fixing Typos

### PR DESCRIPTION
**Pull request by laxmareddyp**
# Description

- [ ] Documentation update

## Checklist

- [ ] I have signed the [Contributor License Agreement](https://github.com/tensorflow/models/wiki/Contributor-License-Agreements).
- [ ] I have read [guidelines for pull request](https://github.com/tensorflow/models/wiki/Submitting-a-pull-request).
- [ ] My code follows the [coding guidelines](https://github.com/tensorflow/models/wiki/Coding-guidelines).
- [ ] I have performed a self [code review](https://github.com/tensorflow/models/wiki/Code-review) of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.

### CODE REVIEW

File: official/vision/ops/augment_test.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""Tests for autoaugment."""
+"""Tests for auto augment."""
```
Comment (fyangf) [inline official/vision/ops/augment_test.py:15]: Do not separate. It is a special op name.

File: official/vision/ops/augment_test.py [hunk 2]
```diff
@@ -297,7 +297,7 @@ def _generate_test_policy(self):
     return policy
 
   def test_custom_policy(self):
-    """Test autoaugment with a custom policy."""
+    """Test auto augment with a custom policy."""
```
Comment (fyangf) [inline official/vision/ops/augment_test.py:300]: ditto

File: official/vision/ops/augment_test.py [hunk 3]
```diff
@@ -311,7 +311,7 @@ def test_custom_policy(self):
        'sub_policy': ('Equalize', 0.9, 11), 'value': '11'},
   )
   def test_invalid_custom_sub_policy(self, sub_policy, value):
-    """Test autoaugment with out-of-range values in the custom policy."""
+    """Test auto augment with out-of-range values in the custom policy."""
```
Comment (fyangf) [inline official/vision/ops/augment_test.py:314]: ditto

File: official/vision/ops/augment_test.py [hunk 4]
```diff
@@ -324,7 +324,7 @@ def test_invalid_custom_sub_policy(self, sub_policy, value):
       augmenter.distort(image)
 
   def test_invalid_custom_policy_ndim(self):
-    """Test autoaugment with wrong dimension in the custom policy."""
+    """Test auto augment with wrong dimension in the custom policy."""
```
Comment (fyangf) [inline official/vision/ops/augment_test.py:327]: ditto

File: official/vision/ops/augment_test.py [hunk 5]
```diff
@@ -335,7 +335,7 @@ def test_invalid_custom_policy_ndim(self):
       augment.AutoAugment(policies=policy)
 
   def test_invalid_custom_policy_shape(self):
-    """Test autoaugment with wrong shape in the custom policy."""
+    """Test auto augment with wrong shape in the custom policy."""
```
Comment (fyangf) [inline official/vision/ops/augment_test.py:338]: ditto

File: official/vision/ops/augment_test.py [hunk 6]
```diff
@@ -345,7 +345,7 @@ def test_invalid_custom_policy_shape(self):
       augment.AutoAugment(policies=policy)
 
   def test_invalid_custom_policy_key(self):
-    """Test autoaugment with invalid key in the custom policy."""
+    """Test auto augment with invalid key in the custom policy."""
```
Comment (fyangf) [inline official/vision/ops/augment_test.py:348]: ditto

**Review by yeqingli — APPROVED** [review]

**Pull request merged**